integration_tests/test_domestic_migration.py

```diff
@@ -114,43 +123,126 @@ def test_individuals_move_into_existing_households(simulants_on_adjacent_timeste
         assert (
             after.loc[
                 mover_to_household_without_reference_person, "relation_to_household_head"
-            ].isin(["Other nonrelative", "Reference person"])
+            ].isin(["Other nonrelative", "Reference person", "Roommate"])
```

Comment:
This kind of comes down to whether we are testing the input data or not. I think we probably should be, and if the input data changes, this test should fail. Of course we wouldn't notice if "Roommate" was removed as an option, but it would at least fail if someone added another option to the file unintentionally.

Reply:
I see your point. Maybe we should do something as simple as defining a variable `other_nonrelative_can_update_to = ["Reference person", "Other nonrelative", "Roommate"]` just to be explicit about what is special about these three relationships.